### DISCUSSION TITLE
Welcome to bagisto Discussions!

### DISCUSSION THREAD

**Discussion by jitendra-webkul** [bagisto/bagisto#5116] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devansh-webkul** [comment]
:rocket:

**Reply by sunteam97** [reply]
![image](https://github.com/user-attachments/assets/89823065-0e9f-4739-9217-d872d6318baa)

Please make popup in product variations
